@navikt/core/react/src/layout/box/Box.tsx

```diff
@@ -149,18 +117,27 @@ export const Box: OverridableComponent<BoxProps, HTMLDivElement> = forwardRef(
         false,
         ["0"],
       ),
-      ...getResponsiveProps("box", "padding", "spacing", padding),
-      ...getResponsiveProps("box", "padding-inline", "spacing", paddingInline),
-      ...getResponsiveProps("box", "padding-block", "spacing", paddingBlock),
     };
 
+    const Comp = asChild ? Slot : Component;
+
     return (
-      <Component
-        {...rest}
-        className={cl("navds-box", className)}
-        ref={ref}
-        style={style}
-      />
+      <BasePrimitive {...rest}>
+        <Comp
+          {...rest}
```

Comment:
double {...rest} ?

Reply:
`BasePrimitive` does not spread props internally, so it only consumes the possible props. So instead of manually assigning every possible prop for BaseProps, a rest spread shortens it. But will need to check that none of the props are passed into html and not filtered out by react. So might need to manually declare each prop anyways...